crates/storage/src/transactional.rs

```diff
@@ -198,7 +265,14 @@ where
 impl<Storage> Modifiable for InMemoryTransaction<Storage> {
     fn commit_changes(&mut self, changes: Changes) -> StorageResult<()> {
         for (column, value) in changes.into_iter() {
-            let btree = self.changes.entry(column).or_default();
+            let btree = match self.changes.entry(column) {
+                hash_map::Entry::Vacant(vacant) => {
+                    vacant.insert(value);
+                    continue
+                }
```

Comment:
If `BTreeMap` is not exist, we just set it to be `value` because it can't have any conflicts=)